core/merginapi.cpp

```diff
@@ -2120,6 +2144,34 @@ void MerginApi::getUserInfoFinished()
   r->deleteLater();
 }
 
+void MerginApi::getSubscriptionInfoFinished()
+{
+  QNetworkReply *r = qobject_cast<QNetworkReply *>( sender() );
+  Q_ASSERT( r );
+
+  if ( r->error() == QNetworkReply::NoError )
+  {
+    CoreUtils::log( "subscription info", QStringLiteral( "Success" ) );
+    QJsonDocument doc = QJsonDocument::fromJson( r->readAll() );
+    if ( doc.isObject() )
+    {
+      QJsonObject docObj = doc.object();
+      mUserInfo->setSubscriptionInfoFromJson( docObj );
```

Comment:
hmm, this is not good. we should have CLEAR separatarion what is get from which endpoint. so what you get from 
login endpoint should be 1 or 2 structs, what you get from subscripion endpoint should be separate struct. ideally named mUserSubscriptionInfo